src/web5-connect/web5-connect-server.ts

```diff
@@ -29,16 +31,30 @@ export type SetWeb5ConnectRequestResult = {
  * The Web5 Connect Server is responsible for handling the Web5 Connect flow.
  */
 export class Web5ConnectServer {
+  public static readonly ttlInSeconds = 600;
 
   private baseUrl: string;
-  private dataStore = new Map(); // TODO: turn this into a TTL cache (https://github.com/TBD54566975/dwn-server/issues/138)
+  private cache: SqlTtlCache;
 
   /**
    * Creates a new instance of the Web5 Connect Server.
    * @param params.baseUrl The the base URL of the connect server including the port.
```

Comment:
nit: add `sqlTtlCacheUrl` here to the params.

Reply:
Done.